docs/sources/setup-grafana/configure-security/configure-authentication/_index.md

```diff
@@ -19,31 +19,25 @@ Grafana provides many ways to authenticate users. Some authentication integratio
-| [Grafana.com OAuth]({{< relref "./grafana-cloud" >}}) | no                | no           | yes          | no                    | N/A       | N/A            | N/A         | yes                  | yes        | yes           |
-| [Okta OAuth]({{< relref "./okta" >}})                 | yes               | yes          | yes          | yes                   | yes       | yes            | N/A         | yes                  | yes        | yes           |
-| [SAML]({{< relref "./saml" >}}) (Enterprise only)     | yes               | yes          | yes          | yes                   | yes       | yes            | N/A         | yes                  | yes        | yes           |
-| [LDAP]({{< relref "./ldap" >}})                       | yes               | yes          | yes          | yes                   | yes       | yes            | yes         | no                   | N/A        | N/A           |
-| [JWT Proxy]({{< relref "./jwt" >}})                   | no                | yes          | yes          | yes                   | no        | no             | N/A         | no                   | N/A        | N/A           |
-
-N/A = Not applicable
-
-## Auth Proxy
-
-| Feature           | Supported? |
-| :---------------- | :--------- |
-| Multi Org Mapping | no         |
-| Enforce Sync      | N/A        |
-| Role Mapping      | yes        |
-
-**Multi Org Mapping:** Able to add a user and role map him to multiple orgs
+| Authentication method                                  | Multi Org Mapping | Enforce Sync | Role Mapping | Grafana Admin Mapping | Team Sync | Allowed groups | Active Sync | Skip OrgRole mapping | Auto Login | Single Logout |
+| :----------------------------------------------------- | :---------------- | :----------- | :----------- | :-------------------- | :-------- | :------------- | :---------- | :------------------- | :--------- | :------------ |
+| [Anonymouse access]({{< relref "./anonymous-auth" >}}) | N/A               | N/A          | N/A          | N/A                   | N/A       | N/A            | N/A         | N/A                  | N/A        | N/A           |
+| [Auth Proxy]({{< relref "./auth-proxy" >}})            | no                | yes          | yes          | no                    | yes       | no             | N/A         | no                   | N/A        | N/A           |
+| [Azure AD OAuth]({{< relref "./azuread" >}})           | yes               | yes          | yes          | yes                   | yes       | yes            | N/A         | yes                  | yes        | yes           |
+| [Basic auth]({{< relref "./grafana" >}})               | yes               | N/A          | yes          | yes                   | N/A       | N/A            | N/A         | N/A                  | N/A        | N/A           |
```

Comment:
Can someone please check if I got the feature availability for basic auth correct?